deployConfig/config.ts

```diff
@@ -297,6 +302,35 @@ const kovanConfig = {
       // },
     ],
   },
+  aaveBorrowConfig: {
```

Comment:
Is aaveBorrowConfig used? On mobile but it looks like the deploy script re-uses existing aaveConfig?